docs/user-manual/wildcard-syntax.adoc

```diff
@@ -72,3 +72,13 @@ For that, the `<wildcard-addresses>` configuration tag is used.
 ----
 
 The example above shows the default configuration.
+
+[NOTE]
+====
+It is technically _possible_ to use `$` as a custom wildcard character, but it is not recommended for two main reasons:
+
+. The default value of `internal-naming-prefix` is `$.activemq.internal.` which includes a `$` character.
+While this value can be changed, doing so may cause problems if, for example, you have already created a cluster while the default value was in use.
+In this case the broker will have already created store-and-forward resources that will have to be renamed manually.
+. The MQTT protocol uses `$` for certain bits of functionality (e.g. shared subscriptions) and this cannot be changed so MQTT functionality would need to be completely avoided.
```

Comment:
As do the AMQP Broker Connection mirroring and federation, possibly others bits.